docs/index.rst

```diff
@@ -6,47 +6,15 @@
 PlasmaPy Documentation
 ######################
 
-.. raw:: html
+.. nbgallery::
+    :name: Examples (frontpage)
+    :glob:
+    :reversed:
+    :hidden:
```

Comment:
It looks like having this here as a replacement for the html markup with examples adds table of content entries to the left on https://plasmapy--792.org.readthedocs.build/en/792/index.html .

This makes sense, as [the `nbgallery` directive seems to be a sort of subclass of `toctree`](https://nbsphinx.readthedocs.io/en/latest/a-normal-rst-file.html?highlight=nbgallery#thumbnail-galleries)

I don't yet know how to hide it from the sidebar, though.

@rocco8773, any ideas?

Reply:
All right, I bruteforced it. :smiling_imp: